sonic-stormond/scripts/stormond

```diff
@@ -309,17 +312,20 @@ class DaemonStorage(daemon_base.DaemonBase):
                 dynamic_kvp_dict["disk_io_reads"] = storage_object.get_disk_io_reads()
                 dynamic_kvp_dict["disk_io_writes"] = storage_object.get_disk_io_writes()
                 dynamic_kvp_dict["reserved_blocks"] = storage_object.get_reserved_blocks()
+                dynamic_kvp_dict["last_sync_time"] = time.time()
```

Comment:
@assrinivasan why not capture simple DATE and time like so in YYYY:MM:DD HH:MM:SS ?

Reply:
Hi @prgeor - the thought process was to keep it consistent with FSSTATS_SYNC time which is captured in seconds since epoch.

Reply:
@assrinivasan can we use the format I mentioned which is more readable than epoch even for sync?

Reply:
FSSTATS_SYNC time is used by the daemon to calculate whether the given interval has passed, to trigger a write to the JSON file. Additionally it isn't user-facing, so it makes sense for us to keep it in epoch format IMO.

Reply:
Added logic to convert time back and forth between epoch and user-friendly format in the latest commit.